src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "Establish your current state" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+import cxSegmentWhitelistInvestigation from 'images/cx_screenshot-full_segment-whitelist-investigation.webp'
+
+import cxQfDashboard from 'images/cx_screenshot-full_qf-dashboard.webp'
+
+import cxQfKpiProgress from 'images/cx_diagram_qf_kpi_progress.webp'
+
+The first step in any improvement experience is to establish your baselines so that you have numbers to measure your KPIs against and can track improvement over time, and improving your customer experience is no exception to this rule. Once you check to make sure everything you need is reporting correctly to our platform, you can validate your information and use our quality foundation dashboard to measure your customer experience.
+
+<Steps>
+    <Step>
+### Review instrumented pages
+
+First, you'll need to review your browser apps and pages to make sure that everything you expect to report to New Relic is actually doing so. You can do this by reviewing the **Page Views** tab in the browser monitoring UI or running the following NRQL query:
```

Comment:
Can we link directly to that Page Views tab? If not, we may want to provide a UI path here.

This paragraph feels pretty abrupt to me. Maybe we need a transition sentence that connects the intro from the previous section to this. something like: "Our browser monitoring tools do such and such. The best place to start with improving your digital experience is reviewing your browser dashboards..." or something like that.

Reply:
We can't link to the Page Views tab unfortunately because it is unique to whatever the customer is ingesting, but I can modify the callout to mention where it exists.

Good call on the transition, I've added something to be a more gentle lead-in.